setup_undercloud.yml

```diff
@@ -31,9 +31,15 @@
     - name: set os_install
       set_fact:
         os_install: "{{ (((osp_release|int > 14) and (rhel_version is version('8.0', '<')))|ternary('RHEL 8', ((osp_release|int < 15) and (rhel_version is version ('8.0', '>=')))|ternary('RHEL 7', false))) }}"
+      when: os_install == false
```

Comment:
So if user sets os_install to true because he wants to reimage undercloud, what version of OS is used to reimage (as it depends on os_install variable in https://github.com/redhat-performance/jetpack/blob/master/setup_undercloud.yml#L61). Maybe, best to use another variable like force_reprovision

Reply:
I will change this as you suggested for better clarity to the users.